### DISCUSSION TITLE
How do I generate the POSCAR files in sys_configs from the training data I ran with cp2k?

### DISCUSSION THREAD

**Discussion by Baopidragon1** [deepmodeling/dpdata#590] [Q&A] [answered]
I've run a series of files with cp2k, and now I'm converting them to raw and npy files that deepmd recognizes. What should I do to prepare for the POSCAR files in the sys_configs folder? Should I use dpdata to batch convert them, and how do I do that exactly?

**Comment by robinzyb** [comment, marked as answer]
POSCAR files in sys_config are used for EXPLORATION. it supports lammps and poscar format. If you have other format of structure files, you could convert it using python package `ase` and `pymatgen`

**Reply by Baopidragon1** [reply]
Thanks for your reply. Do I have to use a POSCAR file in the sys_configs folder? I mean if my dataset is run with cp2k and the fp process I will use cp2k, can the files in the sys_configs folder just use the structure files in cp2k format?

**Reply by robinzyb** [reply]
there is nothing to do with the POSCAR. DP-GEN will read and turn them into usable files for lammps and cp2k

**Reply by Baopidragon1** [reply]
Thank you very much for your answer, so where does the data in the sys_configs folder come from? Is it randomly taken data points from the training data?

**Reply by robinzyb** [reply]
It depends on your purpose. These structures are used in EXPLRATION step, which are read by LAMMPS. You can carefully prepare by yourself or just get from training data.

**Reply by Baopidragon1** [reply]
Thank you very much, and I'd like to ask one last question if you don't mind. If I want to get a dp model of the thermal decomposition mechanism of a substance, and the dataset is obtained by running MD at different temperatures with cp2k, do I need to change the relevant parameter commands in the fp process if I use this dataset to run dpgen? Because I see that the cp2k.inp file generated by dpgen does not contain temperature information if no constraints are added. Or is the fp process just to run a single point of energy for a certain frame structure?

**Reply by robinzyb** [reply]
Nothing need to change when doing fp. fp only provide DFT forces and energies, which has nothing to do with MD evolution. The dp models can be used under different MD condition (npt nvt nve etc.) 
Please look into the Statistical Mechanics by Mark Tuckerman to understand the principle of MD simulation.
